demos/iperf3/kind/README.md

```diff
@@ -1,231 +1,190 @@
 # <ins>Scenario with iPerf3 Connectivity between 2 clusters<ins>
-In this test we check iPerf3 connectivity between different Kind cluster using the MBG components.  
+In this test we check iPerf3 connectivity between different Kind cluster using ClustrLink.  
 This setup use two Kind clusters- 
 1) Cluster 1 - contains the gateway (Control-plane/Data-plane) and iPerf3 client.
 2) Cluster 2 - contains the gateway (Control-plane/Data-plane) and iPerf3 server.
 
-
 ## <ins> Pre-requires installations <ins>
 To run a Kind test, check all pre-requires are installed (Go, docker, Kubectl, Kind):
 
-    export PROJECT_FOLDER=`git rev-parse --show-toplevel`
-    cd $PROJECT_FOLDER
+    export PROJECT_DIR=`git rev-parse --show-toplevel`
+    cd $PROJECT_DIR
     make prereqs
 
 Following are the steps to run the iperf3 connectivity scenario (or you could also run the scripts towards the end):
 ### <ins> Step 1: Build docker image <ins>
-Build MBG docker image:
+Build project docker image:
     
-    make build
     make docker-build
 
 Install local control (gwctl) for the gateway:
     
     sudo make install
 
-### <ins> Step 2: Create Kind clusters with the gateway <ins>
-In this step, we build the Kind cluster with the gateway image.  
+### <ins> Step 2: Create Kind clusters with Clusterlink gateway <ins>
+In this step, we build the Kind cluster with the gateway image.
+Create certificates and deployments files:
+1) Create folder for all deployments and yaml files:
+
+        export TEST_DIR=$PROJECT_DIR/bin/test/iperf3/
+        mkdir -p $TEST_DIR
+        cd $TEST_DIR
+
+2) Create fabric certificates (includes the Root CA certificate fo all the gateways):
+
+        $PROJECT_DIR/bin/cl-adm  create fabric
```

Comment:
nit: there seems to be an extra space between `cl-adm` and the next word (here and below as well)